src/main/kotlin/com/lykke/trade/volume/monitoring/service/notification/SentNotificationsCacheImpl.kt

```diff
@@ -0,0 +1,34 @@
+package com.lykke.trade.volume.monitoring.service.notification
+
+import com.lykke.trade.volume.monitoring.service.config.Config
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+import java.util.*
+import java.util.concurrent.ConcurrentHashMap
+
+@Component
+class SentNotificationsCacheImpl(config: Config): SentNotificationsCache {
```

Comment:
It is better to pass only used parameter values to bean but not whole config

Reply:
fixed